### DISCUSSION TITLE
Adding multiple tokens for users

### DISCUSSION THREAD

**Discussion by UltraSoftware0** [spatie/laravel-google-calendar#228] [Q&A]
Is it possible for me to let multiple users use the token for their google accounts

**Comment by tomcodes** [comment]
Yes it is, see here: https://github.com/spatie/laravel-google-calendar/discussions/179#discussioncomment-3373280

**Reply by Inamulha07** [reply]
Service accounts cannot invite attendees without Domain-Wide Delegation of Authority.
i face this problem
